### PR TITLE
Update VLC bug tracker from Trac to GitLab

### PR DESCRIPTION
**Pull request by TimGels**
### Description of Change ###

Changed Trac to GitLab as bug tracker for the native code in the readme.

### Issues Resolved ### 

N/A

### API Changes ###

 
 None

### Platforms Affected ### 

N/A

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

N/A

### PR Checklist ###

- [ ] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ You have encountered an issue and wish to report it to the VLC dev team?
 
 You can create one on our [Gitlab](https://code.videolan.org/videolan/LibVLCSharp/issues) (using an issue template) for .NET related questions/problems.
 
-_For LibVLC/VLC specific issues, usually native code, trac is our [bug tracker](https://trac.videolan.org/vlc/)._
+_For LibVLC/VLC specific issues, usually native code, VLC GitLab is our [bug tracker](https://code.videolan.org/videolan/vlc/-/issues)._
```
Comment (jeremyVignelles) [inline README.md:311]: I'd rephrase it as GitLab is not a bug tracker per se. What about "native code, please go to the [bug tracker of the VLC project]"?

Comment (TimGels) [reply README.md:311]: Sounds good! I will change it.

**Comment by mfkl** [comment]
Thanks!
